gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -70,6 +72,29 @@
     this.channel = ghfs.getGcsFs().open(gcsPath, readOptions);
   }
 
+  /**
+   * Constructs an instance of GoogleHadoopFSInputStream object with item info
+   *
+   * @param ghfs
```

Comment:
Can we add descriptions for the arguments in javadoc ?

Reply:
I will add descriptions for arguments in the javadoc